File: armstrong.py
n=int(input("Enter a no."))
sum=0
number=n
while(n!=0):
    rem=n%10
    sum=sum+rem*rem*rem
    n//=10
if(sum==number):
    print("Armstrong no.")
else:
    print("not an armstrong no.")
    

File: armstrong.c
#include<stdio.h>
int main()
{
    int n,sum=0,n1=0,r=0;
    printf("Enter a number\n");
    scanf("%d",&n);
    n1=n;
    while(n>0)
    {
      r=n%10;
      sum=sum+(r*r*r);
      n=n/10;
    }
    if(sum==n1)
    printf("%d is an armstrong number",n1);
    else
    printf("%d is not an armstrong number",n1);
    return 0;
}